client/post-editor/post-editor.jsx

```diff
@@ -245,6 +247,10 @@ export const PostEditor = React.createClass( {
 		}
 	},
 
+	toggleNestedSidebar: function( nestedSidebar ) {
```

Comment:
I might be missing something here but to me it feels as though `toggle` is the wrong prefix - instead, `set` seems to be more apt.
I say this because I would expect `toggleX` to be a flip of some sort. Call it once and it sets to one value, call it again and it sets it back to the original value.

The function body is taking any value and setting some state, it just so happens that the usage of this function currently only sees it take 2 different, opposing values. It could easily be used to set a third value at any point, in which case toggle would no longer make sense.